### ISSUE TITLE
Client should only be created once

### ISSUE THREAD

**Issue by Spaceface16518** [cubetastic33/toborrights#6]
Extract it into a single ts file.

**Comment by cubetastic33** [comment]
Now that I combined everything to one page again it is only in one file. Unless the pilot dashboard is gonna create a client again I think it's fine. Is the pilot page gonna do that?

**Comment by Spaceface16518** [comment]
yes it will. and even if it wasn't, it's good to keep it modular. it'll also be nice to implement #3 at the same time. i can take care of this if you want.

**Comment by Spaceface16518** [comment]
this was not fixed. but i can fix it later.
